spec/wasabi/document/encoded_endpoint_spec.rb

```diff
@@ -1,13 +1,13 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe Wasabi::Document do
-  context "with: encoded_endpoint.wsdl" do
-
-    subject { Wasabi::Document.new fixture(:encoded_endpoint).read }
+  context 'with: encoded_endpoint.wsdl' do
+    let(:document) { Wasabi::Document.new fixture(:encoded_endpoint).read }
 
     describe '#endpoint' do
-      subject { super().endpoint }
-      it { should == URI("http://localhost/soapservice/execute?path=/base/includes/Test+Soap/Return+Rows") }
+      subject(:endpoint) { document.endpoint }
+
+      it { should == URI('http://localhost/soapservice/execute?path=/base/includes/Test%20Soap/Return%20Rows') }
```

Comment:
```suggestion
      it { is_expected.to eq(URI('http://localhost/soapservice/execute?path=/base/includes/Test%20Soap/Return%20Rows')) }
```

Perhaps we can use the expect syntax here, too?